front/app/components/IdeasMap/index.tsx

```diff
@@ -427,6 +429,17 @@ const IdeasMap = memo<Props>(
                   });
                 }
               }
+            } else if (topElement.type === 'media') {
+              // If the user clicked on a media layer (E.g. image overlay), show the idea popup
+              if (ideaPostingEnabled) {
+                showAddInputPopup({
```

Comment:
Maybe we can put this into a function? We're adding the same arguments thrice in short succession.